storage/google/cloud/storage/bucket.py

```diff
@@ -386,21 +386,79 @@ class Bucket(_PropertyMixin):
     This is used in Bucket.delete() and Bucket.make_public().
     """
 
+    STANDARD_STORAGE_CLASS = "STANDARD"
+    """Storage class for objects accessed more than once per month."""
+
+    NEARLINE_STORAGE_CLASS = "NEARLINE"
+    """Storage class for objects accessed at most once per month."""
+
+    COLDLINE_STORAGE_CLASS = "COLDLINE"
+    """Storage class for objects accessed at most once per year."""
+
+    MULTI_REGIONAL_LEGACY_STORAGE_CLASS = "MULTI_REGIONAL"
+    """Legacy storage class.
+
+    Alias for :attr:`STANDARD_STORAGE_CLASS`.
+
+    Implies :attr:`MULTI_REGION_LOCATION_TYPE` for :attr:`location_type`.
+    """
+
+    REGIONAL_LEGACY_STORAGE_CLASS = "REGIONAL"
+    """Legacy storage class.
+
+    Alias for :attr:`STANDARD_STORAGE_CLASS`.
+
+    Implies :attr:`REGION_LOCATION_TYPE` for :attr:`location_type`.
+    """
+
+    DURABLE_REDUCED_AVAILABILITY_LEGACY_STORAGE_CLASS = "DURABLE_REDUCED_AVAILABILITY"
+    """Legacy storage class.
+
+    Similar to :attr:`NEARLINE_STORAGE_CLASS`.
+    """
+
     _STORAGE_CLASSES = (
-        "MULTI_REGIONAL",
-        "REGIONAL",
-        "NEARLINE",
-        "COLDLINE",
-        "STANDARD",  # alias for MULTI_REGIONAL/REGIONAL, based on location
-        "DURABLE_REDUCED_AVAILABILITY",  # deprecated
+        STANDARD_STORAGE_CLASS,
+        NEARLINE_STORAGE_CLASS,
+        COLDLINE_STORAGE_CLASS,
+        MULTI_REGIONAL_LEGACY_STORAGE_CLASS,  # deprecated
+        REGIONAL_LEGACY_STORAGE_CLASS,  # deprecated
+        DURABLE_REDUCED_AVAILABILITY_LEGACY_STORAGE_CLASS,  # deprecated
     )
     """Allowed values for :attr:`storage_class`.
 
+    Default value is :attr:`STANDARD_STORAGE_CLASS`.
+
     See
     https://cloud.google.com/storage/docs/json_api/v1/buckets#storageClass
     https://cloud.google.com/storage/docs/storage-classes
     """
 
+    MULTI_REGION_LOCATION_TYPE = "MULTI_REGION"
```

Comment:
"MULTI_REGION", "REGION", and" DUAL_REGION" were supposed to be "multi-region", "region", and "dual-region". This will be changed in the API after a rollout on June 26 and should also be reflected here.